packages/dual-channel/src/app/components/embedded-items/index.js

```diff
@@ -72,7 +72,7 @@ const Container = styled.div`
     z-index: ${styles.zIndex.embeddedItem};
     width: ${mockup.itemWidth.mobile};
     ${props =>
-      props.sectionsPosition !== Waypoint.inside ? '' : `padding-bottom: 100%`};
+      props.sectionsPosition !== Waypoint.inside ? '' : `padding-bottom: 50vh`};
```

Comment:
Since the height of `Container` is not related to its `width` anymore, we can use

```
height: ${props => props.sectionsPosition !== Waypoint.inside ? '0' : '50vh'};
```

to replace setting `padding-bottom`.